tests/performance/performance_tests.cpp

```diff
@@ -21,6 +21,11 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
+
+#if defined(BOOST_TEST_DYN_LINK)
+    #define BOOST_TEST_MAIN
```

Comment:
I usually build against dynamic boost libraries simply by setting `-D Boost_USE_STATIC_LIBS=OFF` on the `cmake` command line. No further modifications are required.

If this is necessary, please use `BOOST_TEST_MODULE` instead, as in https://github.com/bitshares/bitshares-core/blob/master/tests/cli/main.cpp#L58

Reply:
I have been able to compile with the mentioned define and by moving the custom test initialisation routine to the start of the file.